### PR TITLE
Add GuestView

### LINKED ISSUE #204: Handle guest account warning
### Problem

When logging into the guest account we get this rather unpleasant looking dialog

Screenshot from 2023-04-03 12 13 14

### Proposal

Handle the guest account warning here so we can style it nicely and have it be part of the regular onboarding process

### Prior Art (Optional)

_No response_

### PR DESCRIPTION
**Pull request by danirabbit**
Fixes #204
Fixes https://github.com/elementary/triage/issues/144

Screenshot from 2023-04-04 17 35 52

### CODE REVIEW

**Review by lenemter — COMMENTED** [review]
Online account are deleted once logged out, right? So I think we shouldn't show them in guest sessions?

File: src/Views/GuestView.vala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                max_width_chars = 40,
+                wrap = true,
+                xalign = 0
+            };
+
+            spacing = 6;
+            append (image);
+            append (description_label);
+        }
+    }
```
Comment (lenemter) [inline src/Views/GuestView.vala:69]: This class is already defined in Early Access view, maybe reuse it instead of copying code over here?

**Comment by danirabbit** [comment]
@lenemter Yeah I'm on the fence on that one. If you're logging in to the guest account to check your email I think maybe it still makes sense to point to where to set up online accounts?

**Comment by lenemter** [comment]
@danirabbit Well, maybe. But IMO it's easier to use web browser than adding an online account if you're using guest session. Anyway, I don't confront your decision to show online accounts here.

**Comment by danirabbit** [comment]
Yeah that's a good point. I'll remove it and we can always add it back in later if people make noise

**Review by lenemter — APPROVED** [review]
Looks good. Nice work :tada:

**Pull request merged**